tests/playlists.py

```diff
@@ -86,6 +86,18 @@
 http://media.example.com/fileSequence52-3.ts
 '''
 
+PLAYLIST_WITH_CUSTOM_MEDIA_HEADER = '''#EXTM3U
```

Comment:
I think you can name it `PLAYLIST_WITH_CUSTOM_TAG` because your code fixes every custom tag, not just `EXT-X-MEDIA`. Do you agree?